hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/TestStateContext.java

```diff
@@ -126,6 +126,12 @@ public void testPutBackReports() {
       fail("Should throw exception when putting back unaccepted reports!");
     } catch (IllegalArgumentException ignored) {
     }
+    try {
```

Comment:
can you elaborate the putback crl report scenario below?

Reply:
Yes. Some reports like incremental container reports support putback, but we don't want to put back the CRL status report in any scenario. Even if the report doesn't get delivered to SCM, a new report will be generated in the next heartbeat. This test is to affirm that an exception is thrown while putting back the CRL status report.